scss/bitstyles/base/html/html.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
```

Comment:
Nice ✨ 

Didn't know this was possible.

On that note: does it make sense to apply a certain order to the sidebar in Storybook?